mesheryctl/internal/cli/root/mesh/validate.go

```diff
@@ -102,11 +102,11 @@ var validateCmd = &cobra.Command{
 }
 
 func init() {
-	validateCmd.Flags().StringVarP(&spec, "spec", "s", "smi", "specification to be used for conformance test")
-	_ = validateCmd.MarkFlagRequired("spec")
-	validateCmd.Flags().StringVarP(&adapterURL, "adapter", "a", "meshery-osm:10010", "Adapter to use for validation")
-	_ = validateCmd.MarkFlagRequired("adapter")
-	validateCmd.Flags().StringVarP(&namespace, "namespace", "n", "default", "Kubernetes namespace to be used for deploying the validation tests and sample workload")
-	validateCmd.Flags().StringVarP(&tokenPath, "tokenPath", "t", "", "Path to token for authenticating to Meshery API")
-	_ = validateCmd.MarkFlagRequired("tokenPath")
+	ValidateCmd.Flags().StringVarP(&spec, "spec", "s", "smi", "specification to be used for conformance test")
+	_ = ValidateCmd.MarkFlagRequired("spec")
+	ValidateCmd.Flags().StringVarP(&adapterURL, "adapter", "a", "meshery-osm:10010", "Adapter to use for validation")
+	_ = ValidateCmd.MarkFlagRequired("adapter")
+	ValidateCmd.Flags().StringVarP(&namespace, "namespace", "n", "default", "Kubernetes namespace to be used for deploying the validation tests and sample workload")
+	ValidateCmd.Flags().StringVarP(&tokenPath, "tokenPath", "t", "", "Path to token for authenticating to Meshery API")
```

Comment:
Use of `utils.AuthConfigFile` (example here - https://github.com/layer5io/meshery/blob/49819bd9438140cc7ad5572716c1ae2c158e511c/mesheryctl/internal/cli/root/perf/perf.go#L161) seems to provide CLI users with the convenience of assuming a default location for the token file. Is that something we might incorporate here? If so, consistency of UX is ideal.

Reply:
Yea I think so

Reply:
Although the change for this would be done in another PR where I've made changes

Reply:
Ok, sounds good.